### DISCUSSION TITLE
关于获取accessToken:请求失败invalid appsecret rid xxxxx

### DISCUSSION THREAD

**Discussion by wangxinleo** [wangxinleo/wechat-public-account-push#68] [Q&A]
首次登陆微信测试号，第一次生成的appsecret可能无法使用，重新登陆进行刷新获取正常的appsecret即可。

**Comment by liangyao1** [comment]
怎么刷新啊  测试号登陆过一次就没有变化了  没找到刷新按钮

**Reply by wangxinleo** [reply]
重新扫码登陆。

**Comment by Oicite** [comment]
好像还是有错 作者  我设置了好多遍还是报这个

**Comment by iuabvgegw7076** [comment]
我也遇到了这个问题,解决方式并不是刷新,而是因为index.cjs的useTemplateId填错了,填微信公众号上的模版id就对了
